openedx/core/djangoapps/user_authn/views/logout.py

```diff
@@ -105,6 +110,18 @@ def _is_enterprise_target(self, url):
         unquoted_url = parse.unquote_plus(parse.quote(url))
         return bool(re.match(r'^/enterprise/[a-z0-9\-]+/course', unquoted_url))
 
+    def _show_tpa_logout_link(self, target, referrer):
+        """
+        Return Boolean value indicating if TPA logout link needs to displayed or not
```

Comment:
Can you specify in the comment, under what condition do we want to display the TPA logout link?

Reply:
@schenedx  Updated docstring with condition.